### DISCUSSION TITLE
NR01 for gopro 10

### DISCUSSION THREAD

**Discussion by DrkAlexDrk** [gopro/labs#615] [Feature Requests]
Hello,

The performance of the GoPro in low light conditions is quite poor (like any camera with such a small sensor).
However, on YouTube you can see videos of people who have managed to record scenes with their GoPro 11 with excellent quality, in low lighting conditions. To do this, they have relied on the options provided by GoPro Labs. Specifically, they have used the NR01 commands to post-process the noise with other PC software, WIDE and LOGB. I understand (am I wrong?) that these last two commands cannot apply to GoPro 10, due to the difference in the sensor (is there something similar for the GoPro 10?).
However, I do not understand why the NR01 command cannot be applied to the GoPro 10, taking into account that it also has noise cancellation algorithms.

Is there any plan to add any of these features to the GoPro 10 firmware?

thank you so much

**Comment by dnewman-gpsw** [comment]
Both WIDE and LOGB require 10-bit video, as in 8-bit modes they introduce unwanted banding, so this is reason they aren't easily back ported to HERO10.  NR01 might be possible on HERO10.  I will add it to wish list to see if it can happen. A hoping another HERO10 Labs build might happen early next year (not any sooner.)

**Reply by kouyoumin** [reply]
WIDE/LOGB and NR01 should be a good combination because noise mitigates banding (just like dithering.)

**Reply by tco7878** [reply]
Is there any update on this? It would be awesome to be able to disable noise reduction entirely on the Hero10.

**Reply by dnewman-gpsw** [reply]
HERO10 did get a Labs update, but the denoise hack was too deep to be back-ported.  Denoise control is now standard on HERO13, and Labs only for HERO11/Mini and HERO12.

**Reply by Rabii-Hayani** [reply]
Gh 12 deserve to had a officiel firmware update to add the denoise in video 
The same GP2 the same sensor ....

**Reply by dnewman-gpsw** [reply]
That is not a Labs question, HERO12 does have denoise control through Labs.

**Reply by Rabii-Hayani** [reply]
But all the new features the team test its in the labs firmware .. then they push it to the officiel firmware .... as a new tested features 
Hero 12 black deserve some features in the hero 13 black 
Any way thx david 👍👍

**Reply by dnewman-gpsw** [reply]
That is not how it works. 95% of feature in Labs never get ported to mainline. The 5% of features that move, and rarely ported in code, there written from scratch for quality. Today only Labs has full control over denoise, even on HERO13.
